src/main/scala/purecsv/safe/package.scala

```diff
@@ -66,12 +68,12 @@ package object safe {
     def readCSVFromReader(r: Reader,
                           delimiter: Char = RecordSplitter.defaultFieldSeparator,
                           trimming: Trimming = NoAction,
-                          skipHeader: Boolean = false
-                          ): Iterator[Try[A]] = {
+                          skipHeader: Boolean = false,
+                          fields: List[String]): Iterator[Try[A]] = {
```

Comment:
with fields - can we call ClassUtil inside this function to avoid having the calling code make the call - if not, can we default this to Seq.empty? I'd prefer to use Seq over List.

Reply:
eg like in line 98 below